sdk/face/ai-vision-face-rest/package.json

```
@@ -0,0 +1,143 @@
+{
+  "name": "@azure-rest/ai-vision-face",
+  "version": "1.0.0-beta.1",
+  "description": "Face API REST Client",
+  "engines": {
+    "node": ">=18.0.0"
+  },
+  "sideEffects": false,
+  "autoPublish": false,
+  "tshy": {
+    "exports": {
+      "./package.json": "./package.json",
+      ".": "./src/index.ts"
+    },
+    "dialects": [
+      "esm",
+      "commonjs"
+    ],
+    "esmDialects": [
+      "browser",
+      "react-native"
+    ],
+    "selfLink": false
+  },
+  "type": "module",
+  "keywords": [
+    "node",
+    "azure",
+    "cloud",
+    "typescript",
+    "browser",
+    "isomorphic"
+  ],
+  "author": "Microsoft Corporation",
+  "license": "MIT",
+  "files": [
+    "dist",
+    "README.md",
+    "LICENSE",
+    "review/*"
+  ],
+  "sdk-type": "client",
+  "repository": "github:Azure/azure-sdk-for-js",
+  "bugs": {
+    "url": "https://github.com/Azure/azure-sdk-for-js/issues"
+  },
+  "homepage": "https://github.com/Azure/azure-sdk-for-js/tree/main/sdk/face/ai-vision-face-rest/README.md",
+  "prettier": "@azure/eslint-plugin-azure-sdk/prettier.json",
+  "//metadata": {
+    "constantPaths": [
+      {
+        "path": "src/faceClient.ts",
+        "prefix": "package-version"
+      }
+    ]
+  },
+  "dependencies": {
+    "@azure-rest/core-client": "^2.0.0",
+    "@azure/core-auth": "^1.6.0",
+    "@azure/core-rest-pipeline": "^1.5.0",
+    "@azure/logger": "^1.0.0",
+    "tslib": "^2.6.2",
+    "@azure/core-lro": "^2.5.4",
+    "@azure/abort-controller": "^1.0.0"
+  },
+  "devDependencies": {
+    "dotenv": "^16.0.0",
+    "@microsoft/api-extractor": "^7.40.3",
+    "@types/node": "^18.0.0",
+    "eslint": "^8.55.0",
+    "prettier": "^3.2.5",
+    "rimraf": "^5.0.5",
+    "mkdirp": "^3.0.1",
+    "typescript": "~5.4.5",
+    "tshy": "^1.11.1",
+    "@azure/core-util": "^1.0.0",
+    "@azure-tools/test-credential": "^1.0.0",
+    "@azure/identity": "^4.0.1",
+    "@azure-tools/test-recorder": "^4.0.0",
+    "@vitest/browser": "^1.3.1",
+    "@vitest/coverage-istanbul": "^1.3.1",
+    "playwright": "^1.41.2",
+    "vitest": "^1.3.1",
+    "@azure/dev-tool": "^1.0.0",
+    "@azure/eslint-plugin-azure-sdk": "^3.0.0"
+  },
+  "scripts": {
+    "clean": "rimraf --glob dist dist-browser dist-esm test-dist temp types *.tgz *.log",
+    "extract-api": "rimraf review && mkdirp ./review && dev-tool run extract-api",
+    "pack": "npm pack 2>&1",
+    "lint": "eslint package.json api-extractor.json src test --ext .ts --ext .cts --ext .mts",
+    "lint:fix": "eslint package.json api-extractor.json src test --ext .ts --ext .cts --ext .mts --fix --fix-type [problem,suggestion]",
+    "unit-test": "npm run unit-test:node && npm run unit-test:browser",
+    "unit-test:browser": "npm run build:test && dev-tool run test:vitest --browser",
```

There are some dependencies not avalibale in browser environment, let's skip browser testing and have a separated pr to fix browser testing issue.

```suggestion
    "unit-test:browser": "echo skipped",
```

---

I’m fixing this locally by disabling tests reading files and implementing a simple randomUUID function. I will push those changes after finishing all the checks.